### PR TITLE
More chain melting

### PR DESCRIPTION
**Pull request by g0urra**

### CODE REVIEW

File: kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js [hunk 1]
```diff
@@ -91,6 +91,41 @@ onEvent('recipes', (event) => {
             temperature: 800,
             time: 58,
             id: 'tconstruct:smeltery/melting/metal/iron/chain'
+        },
+        {
+            ingredient: { item: 'dustrial_decor:gold_chain' },
+            result: { fluid: 'tconstruct:molten_gold', amount: 22 },
+            temperature: 800,
```
Comment (MuteTiefling) [inline kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js:98]: melting points should be consistent with the base material. gold is 700

File: kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js [hunk 2]
```diff
@@ -91,6 +91,41 @@ onEvent('recipes', (event) => {
             temperature: 800,
             time: 58,
             id: 'tconstruct:smeltery/melting/metal/iron/chain'
+        },
+        {
+            ingredient: { item: 'dustrial_decor:gold_chain' },
+            result: { fluid: 'tconstruct:molten_gold', amount: 22 },
+            temperature: 800,
+            time: 58,
+            id: '${id_prefix}molten_gold/chain'
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js:100]: This id is different from the rest, why is that?

Comment (MuteTiefling) [inline kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js:100]: id's should be descriptive of what they make, and if there's a chance of collision, what the source is.

molten metals are a good example of where things can collide, as many things melt down to them. So I'd prefer to see something like this for the ID's:

\`${id_prefix}molten_gold_from_gold_chain\`

You also need to use backticks \` here for the ${id_prefix} to be used properly.

File: kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js [hunk 3]
```diff
@@ -91,6 +91,41 @@ onEvent('recipes', (event) => {
...
+        {
+            ingredient: { item: 'dustrial_decor:gold_chain' },
+            result: { fluid: 'tconstruct:molten_gold', amount: 22 },
+            temperature: 800,
+            time: 58,
+            id: '${id_prefix}molten_gold/chain'
+        },
+        {
+            ingredient: { item: 'betterendforge:thalassium_chain' },
+            result: { fluid: 'tconstruct:molten_thalassium', amount: 22 },
```
Comment (MuteTiefling) [inline kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js:104]: This has made me realize that chains need a bit of work in general. the 22mb used above in iron chains is from the cheapened iron chain recipe. 

8 chains = 11 nuggets and 1 nugget is 16mb

So
8 chains = 176mb or 1 chain = 22mb

Not all of the chain recipes have the same values though. Would you be comfortable altering shaped crafting recipes as well to match iron/gold chains? (one craft gets 8 chains)

**Review by MuteTiefling — CHANGES_REQUESTED** [review]

File: kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js (hunk 2 above)
Comment (g0urra) [reply kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js]: I am actually not sure about it.

File: kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js (hunk 3 above)
Comment (g0urra) [reply kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js]: I am not sure what this refers to? I thought I had made sure that other chains were the same cost, but I'll take another look.

Comment (g0urra) [reply kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js]: I have fixed the values for Thalassium, Terminite and Nebu as they were inconsistent.

File: kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js (hunk 2 above)
Comment (g0urra) [reply kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js]: Fixed!

**Review by NillerMedDild — APPROVED** [review]

**Comment by NillerMedDild** [comment]
Thank you for your contribution 😃

**Pull request merged**